Fixed remaining static errors in CSCDCCUnpacker.cc

uhussain opened the pull request.

uhussain committed:
PackedCandidate closest vertex

uhussain committed:
Fixed 6 remaining staticErrors in CSCDCCUnpacker.cc

cmsbuild commented:
A new Pull Request was created by  @uhussain  for CMSSW_8_1_X.

It involves the following packages:

CommonTools/PileupAlgos
EventFilter/CSCRawToDigi


@cmsbuild, @cvuosalo, @slava77, @davidlange6 can you please review it and eventually sign? Thanks.
@ahinzmann, @jdolen, @rappoccio, @barvic, @Martin-Grunewald this is something you requested to watch as well.
@slava77, @degano, @smuzaffar you are the release manager for this.

cms-bot commands are list here cms-sw/cmssw#13028

uhussain commented:
Please ignore this pull request, it has a change that has already been
incorporated. I am sorry I created this from another branch. I have updated
my current pull request from my CMSSW_8_1_X branch.

Thanks!

On Mon, Apr 25, 2016 at 4:51 PM, cmsbuild <notifications@github.com> wrote:

> A new Pull Request was created by @uhussain <https://github.com/uhussain>
> for CMSSW_8_1_X.
>
> It involves the following packages:
>
> CommonTools/PileupAlgos
> EventFilter/CSCRawToDigi
>
> @cmsbuild <https://github.com/cmsbuild>, @cvuosalo
> <https://github.com/cvuosalo>, @slava77 <https://github.com/slava77>,
> @davidlange6 <https://github.com/davidlange6> can you please review it
> and eventually sign? Thanks.
> @ahinzmann <https://github.com/ahinzmann>, @jdolen
> <https://github.com/jdolen>, @rappoccio <https://github.com/rappoccio>,
> @barvic <https://github.com/barvic>, @Martin-Grunewald
> <https://github.com/Martin-Grunewald> this is something you requested to
> watch as well.
> @slava77 <https://github.com/slava77>, @degano <https://github.com/degano>,
> @smuzaffar <https://github.com/smuzaffar> you are the release manager for
> this.
>
> cms-bot commands are list here cms-sw/cmssw#13028
> <https://github.com/cms-sw/cmssw/issues/13028>
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/cms-sw/cmssw/pull/14244#issuecomment-214538523>
>

cvuosalo commented:
-1
@uhussain; Can you close this PR?
